### PR TITLE
GraphQL v1: accept slugs as query parameters in allCollectives

### PR DESCRIPTION
**Pull request by NotJohs**
… and return collective data.
Re [#2565](https://github.com/opencollective/opencollective/issues/2565)

### CODE REVIEW

File: server/graphql/v1/queries.js [hunk 1]
```diff
@@ -795,6 +795,10 @@ const queries = {
   allCollectives: {
     type: CollectiveSearchResultsType,
     args: {
+      renderCollectivesFromList: {
```
Comment (znarf) [inline server/graphql/v1/queries.js]: should be named `slugs`

File: server/graphql/v1/queries.js [hunk 2]
```diff
@@ -859,6 +863,12 @@ const queries = {
         include: [],
       };
 
+      if (args.renderCollectivesFromList) {
+        query.where.id = await Promise.all(
+          args.renderCollectivesFromList.map(async id => (typeof id === 'string' ? await fetchCollectiveId(id) : id)),
+        );
```
Comment (znarf) [inline server/graphql/v1/queries.js]: Should just be a where `IN` condition with the slugs. As simple as it can be.

Comment (NotJohs) [reply server/graphql/v1/queries.js]: Yes. I just made that change.

File: server/graphql/v1/queries.js (hunk 1 above)
Comment (NotJohs) [reply server/graphql/v1/queries.js]: Understood.

**Pull request merged**